usn/ubuntu_server_netboot.py

```diff
@@ -24,12 +24,12 @@
 
 
 Netboot_Args = ["root=/dev/ram0", "ramdisk_size=1500000", "ip=dhcp"]
+
```

Comment:
This is one of the whitespace changes I'm referring to

Reply:
Removed on the last commit.